vue-frontend/src/components/home/HomeConsultation.vue

```diff
@@ -93,13 +98,18 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.consultant-container {
-  margin: 48px 10%;
+.container-fluid {
+  padding: 0 10%;
+}
+
+.container {
+  padding: 0;
   display: flex;
   flex-direction: column;
   border-radius: 0.9rem;
   border: 1px solid #e2e2e2;
   overflow: hidden;
+  width: 100%;
```

Comment:
Let's never change width of container. If we need to do it, let's add another div as wrapper.